### PR TITLE
Fix: Admin Top Bar - Remove from non Elementor pages [ED-11695]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: modules/admin-top-bar/module.php [hunk 1]
```diff
@@ -88,22 +88,43 @@ private function add_frontend_settings() {
...
 
+	private function should_be_shown() {
+		$current_screen = get_current_screen();
+
+		if ( ! $current_screen ) {
+			return false;
+		}
+
+		$is_elementor_page = strpos( $current_screen->id ?? '', 'elementor' ) !== false;
+		$is_elementor_post_type_page = strpos( $current_screen->post_type ?? '', 'elementor' ) !== false;
```
Comment (StyleShit) [inline modules/admin-top-bar/module.php:99]: hmm, what's gonna happen with 3rd party plugins that uses "elementor" in their name?

File: modules/admin-top-bar/module.php [hunk 2]
```diff
@@ -88,22 +88,43 @@ private function add_frontend_settings() {
...
+
+		return apply_filters(
+			'elementor/admin-top-bar/should-be-shown',
```
Comment (StyleShit) [inline modules/admin-top-bar/module.php:102]: IDK why, but this hook name sound weird ("should be shown"). maybe let's talk about it and try to find a different name?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: modules/admin-top-bar/module.php (hunk 1 above)
Comment (Nevoss) [reply modules/admin-top-bar/module.php:99]: Done

File: modules/admin-top-bar/module.php (hunk 2 above)
Comment (Nevoss) [reply modules/admin-top-bar/module.php]: Done

**Review by StyleShit — APPROVED** [review]

**Pull request merged**